BlueShift-iOS-SDK/BlueShift.m

```diff
@@ -726,7 +724,7 @@ - (void)registerForInAppMessage:(NSString *)displayPage {
 
 - (void)unregisterForInAppMessage {
     if (_inAppNotificationMananger && _config.inAppManualTriggerEnabled == NO) {
```

Comment:
@ketanshikharebsft could you please check if this check on manual mode flag will break the display of in-app in manual mode?

Reply:
I have made the necessary changes for the In-app ManualTrigger.